flixel/system/render/common/FlxCameraView.hx

```diff
@@ -0,0 +1,275 @@
+package flixel.system.render.common;
+
+import flixel.FlxCamera;
+import flixel.graphics.FlxGraphic;
+import flixel.graphics.TrianglesData;
+import flixel.graphics.frames.FlxFrame;
+import flixel.math.FlxMatrix;
+import flixel.math.FlxPoint;
+import flixel.math.FlxRect;
+import flixel.graphics.shaders.FlxShader;
+import flixel.util.FlxColor;
+import flixel.util.FlxDestroyUtil;
+import openfl.display.BitmapData;
+import openfl.display.BlendMode;
+import openfl.display.DisplayObject;
+import openfl.display.Graphics;
+import openfl.filters.BitmapFilter;
+import openfl.geom.ColorTransform;
+import openfl.geom.Point;
+import openfl.geom.Rectangle;
+
+// TODO: add pure opengl version of camera view, so it will work on nme...
+// TODO: add stage3d version of camera view...
+
+/**
+ * ...
+ * @author Zaphod
+ */
+class FlxCameraView implements IFlxDestroyable
+{
+	// Batching related static variables and constants:
+	public static inline var MAX_INDICES_PER_BUFFER:Int = 98298;
+	public static inline var MAX_VERTEX_PER_BUFFER:Int = 65532;		// (MAX_INDICES_PER_BUFFER * 4 / 6)
+	public static inline var MAX_QUADS_PER_BUFFER:Int = 16383;		// (MAX_VERTEX_PER_BUFFER / 4)
+	public static inline var MAX_TRIANGLES_PER_BUFFER:Int = 21844;	// (MAX_VERTEX_PER_BUFFER / 3)
+	
+	public static inline var VERTICES_PER_QUAD:Int = 4;
+	
+	public static inline var TRIANGLES_PER_QUAD:Int = 2;
+	
+	public static inline var INDICES_PER_TRIANGLE:Int = 3;
+	
+	public static inline var INDICES_PER_QUAD:Int = 6;
+	
+	/**
+	 * Max size of the batch. Used for quad render items. If you'll try to add one more tile to the full batch, then new batch will be started.
+	 */
+	public static var QUADS_PER_BATCH(default, set):Int = 2000;
+	
+	private static function set_QUADS_PER_BATCH(value:Int):Int
+	{
+		QUADS_PER_BATCH = (value > MAX_QUADS_PER_BUFFER) ? MAX_QUADS_PER_BUFFER : value;
+		return QUADS_PER_BATCH;
+	}
+	
+	public static var TRIANGLES_PER_BATCH(default, set):Int = 2600;
+	
+	private static function set_TRIANGLES_PER_BATCH(value:Int):Int
+	{
+		TRIANGLES_PER_BATCH = (value > MAX_TRIANGLES_PER_BUFFER) ? MAX_TRIANGLES_PER_BUFFER : value;
+		return TRIANGLES_PER_BATCH;
+	}
+	
+	/**
+	 * Whether to batch drawTriangles() calls or not.
+	 * Default value is true.
+	 */
+	public static var BATCH_TRIANGLES:Bool = #if FLX_RENDER_GL false #else true #end;
+	
+	/**
+	 * Tracks total number of `drawTiles()` calls made each frame.
+	 */
+	public static var _DRAWCALLS:Int = 0;
```

Comment:
I guess this was always called `_DRAWCALLS`? Weird naming, normally we have a leading underscore for private vars, and it's not constant either, so it probably shouldn't be upper-cased...

Maybe a good opportunity to rename it to `drawCalls`?